asyncio_run_in_process/main.py

```diff
@@ -47,6 +48,15 @@
 
 
 logger = logging.getLogger("asyncio_run_in_process")
+_executor: Optional[concurrent.futures.ThreadPoolExecutor] = None
+
+
+def _get_executor() -> concurrent.futures.ThreadPoolExecutor:
+    global _executor
```

Comment:
I'm not a fan usage of `global` but I also don't have an alternative to propose if we want to maintain the current API of this library. So, yeah, nothing to see here :)